platforms/Windows/bundle/installer.wixproj

```diff
@@ -21,6 +21,7 @@
     <ProjectReference Include="..\cli\asserts\cli.asserts.wixproj" BindName="cli.asserts" />
     <ProjectReference Include="..\dbg\asserts\dbg.asserts.wixproj" BindName="dbg.asserts" />
     <ProjectReference Include="..\ide\asserts\ide.asserts.wixproj" BindName="ide.asserts" />
+    <ProjectReference Include="..\python\asserts\python.wixproj" BindName="python" />
```

Comment:
please get rid of `asserts` subdirectory, and just have everything directelly under `python`